packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -762,6 +768,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 
     private processInboundMessage(message: ISequencedDocumentMessage): void {
         const startTime = Date.now();
+        this.opsCurrentlyProcessing++;
```

Comment:
I'd rather make it a boolean (as there is no reentrancy here). You could also add asserts that it is false (zero in current form) before we change it here, and true below we reset it back